libs/application/templates/social-insurance-administration/old-age-pension/src/fields/Conclusion/index.tsx

```diff
@@ -0,0 +1,140 @@
+import {
+  FieldBaseProps,
+  FieldComponents,
+  FieldTypes,
+} from '@island.is/application/types'
+import { Markdown } from '@island.is/shared/components'
+import {
+  Box,
+  AlertMessage,
+  AccordionCard,
+  BulletList,
+} from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { FC } from 'react'
+import { MessageWithLinkButtonFormField } from '@island.is/application/ui-fields'
+import { socialInsuranceAdministrationMessage } from '@island.is/application/templates/social-insurance-administration-core/lib/messages'
+import { getApplicationExternalData } from '../../lib/oldAgePensionUtils'
+import { oldAgePensionFormMessage } from '../../lib/messages'
+import { coreMessages } from '@island.is/application/core'
+
+export const Conclusion: FC<React.PropsWithChildren<FieldBaseProps>> = (
+  props,
+) => {
+  const { application } = props
+  const { formatMessage } = useLocale()
+
+  const { hasIncomePlanStatus } = getApplicationExternalData(
+    application.externalData,
+  )
+
+  if (!hasIncomePlanStatus) {
+    return (
+      <Box marginTop={2}>
+        <Box marginBottom={5}>
+          <AlertMessage
+            type="warning"
+            title={formatMessage(
+              socialInsuranceAdministrationMessage.conclusionScreen
+                .receivedAwaitingIncomePlanTitle,
```

Comment:
Would it not be nicer to just have a return instead of if else? Since most of the JSX is the same (especially the AccordionCard), you could just use a ternary operator (a ? b : c) for the differing lines. Like for the AlertMesage  title you can do 

```
hasIncomePlanStatus
  ? socialInsuranceAdministrationMessage.conclusionScreen.alertTitle
  : socialInsuranceAdministrationMessage.conclusionScreen.receivedAwaitingIncomePlanTitle
```